release/changes.md

```diff
@@ -0,0 +1 @@
+- [NEW] Improve tags/values in GitHub Actions `pull_request` workflows using git detached HEAD state
```

Comment:
Are there options aside from using a Git detached HEAD state? If not, I would just simplify it to:

```suggestion
- [NEW] Improve tags/values in GitHub Actions `pull_request` workflows
```

Reply:
No, it seems it will always be in detached HEAD state from reading the docs.